src/redismodule.rs

```diff
@@ -155,6 +167,19 @@ impl Drop for RedisString {
     }
 }
 
+impl Display for RedisString {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.into_string_lossy())
+    }
+}
+
+impl Borrow<str> for RedisString {
+    fn borrow(&self) -> &str {
+        // TODO remove unwrap()
+        self.try_as_str().unwrap()
```

Comment:
@oshadmi Of course it can -- the TODO should be completed, probably by using `from_utf8_lossy` or by returning a fixed placeholder value (if we want to keep performance known).

Reply:
```
        self.try_as_str().unwrap_or_default()
```
Should do?